src/publish-size.mjs

```diff
@@ -0,0 +1,30 @@
+import { repo, sha, event, branch, pull_request_number, ci } from 'ci-env';
+import axios from 'axios';
+
+const SIZE_STORE_ENDPOINT = process.env.SIZE_STORE_ENDPOINT || 'https://size-plugin-store.now.sh' ;
+
+// TODO: add option to turn off publishing of sizes.
+
+export async function publishDiff(diff,filename) {
+	if (process.env.NODE_ENV !=='test' &&  ci && event == 'pull_request') {
+		try {
+			const params = { ci,repo, branch, sha, pull_request_number, diff,filename };
+			await axios.post(`${SIZE_STORE_ENDPOINT}/diff`, params);
```

Comment:
We could replace axios with a plain `fetch` call here. It doesn't do anything axios specific.

Note, that I'm not a maintainer here so take this with a grain of salt :)